web-server/v0.4/src/components/SiderMenu/index.js

```diff
@@ -9,9 +9,9 @@ const SiderMenuWrapper = props => {
     <DrawerMenu
       getContainer={null}
       level={null}
-      handleChild={<i className="drawer-handle-icon" />}
+      handlechild={<i className="drawer-handle-icon" />}
```

Comment:
This seems like an interface inconsistency with the `DrawerMenu`.  Can you confirm?  The GitHub repo does not seem to have any references to `handlechild`.

Reply:
Yes, absolutely. There is no such method there and we do not use it in our code as well. So removing that.